src/content/apis/social/shrtcode.md

```diff
@@ -0,0 +1,11 @@
+---
+name: Shrtcode API
+description: Free API to shorten Links
+categories:
+  - social
```

Comment:
I don't think if the API fits in the `social` category. It would be best to change it into `miscs` right now.
```suggestion
  - miscs
```